README.md

```diff
@@ -1,2 +1,92 @@
-# website
-Repository hosting the website files (guides, etc).
+# Service Binding
+
+## Introduction
+
+Coming soon.
+
+## Installing Service Binding Controller
+
+[Service Binding Controller](https://github.com/k8s-service-bindings/service-binding-controller) is a Kubernetes operator that implements [Service Binding Specification](https://github.com/k8s-service-bindings/spec).  Service Binding Controller works with the recent versions of Kubernetes.  You can use Kubernetes version 1.19 or above.  You also need [kubectl](https://kubernetes.io/docs/tasks/tools/) CLI tool install in your system.
+
+You can install the latest version of Service Binding Controller with this command:
+
+```
+kubectl apply -f https://github.com/k8s-service-bindings/service-binding-controller/releases/latest/download/release.yaml
+```
+
+If you need a particular version, replace `<version>` in this command:
+
+```
+kubectl apply -f https://github.com/k8s-service-bindings/service-binding-controller/releases/download/<version>/release.yaml
+```
+
+These are the major resources that is getting created during the installation:
+
+1. A namespace with name as `service-binding-controller-system`.  All the namespace scoped resources are created in this namespace.
+2. Two CRDs: `ClusterWorkloadResourceMapping` (cluster scoped), `ServiceBinding` (namespace scoped)
+3. ClusterRole: `service-binding-controller-manager-role-with-aggregation-rule`
```

Comment:
let's remove this until these commands actually work

Reply:
Removed.